webpack.js

```diff
@@ -0,0 +1,35 @@
+const { merge } = require('webpack-merge')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+const BabelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except')
+
+const config = {
+	module: {
+		rules: [
+			{
+				test: /\.vue$/,
+				loader: 'vue-loader',
+				exclude: BabelLoaderExcludeNodeModulesExcept([
+					'@juliushaertl/vue-richtext',
+					'vue-material-design-icons',
+				]),
```

Comment:
Not needed for 1.4.1
https://github.com/nextcloud/webpack-vue-config/blob/25748b4bb202764414b168d1e0c0b90e05eeccf9/webpack.js#L42

Reply:
Removed